be/src/exec/olap_table_sink.cpp

```diff
@@ -640,14 +640,25 @@ void OlapTableSink::_convert_batch(RuntimeState* state, RowBatch* input_batch, R
 #else
                     state->append_error_msg_to_file("", ss.str());
 #endif
-                    continue;
+                    exist_null_value_for_not_null_col = true;
+                    break;
                 }
             }
             void* slot = dst_tuple->get_slot(slot_desc->tuple_offset());
             RawValue::write(src_val, slot, slot_desc->type(), _output_batch->tuple_data_pool());
         }
+
+        if (!exist_null_value_for_not_null_col) {
+            if (output_batch->add_row() != RowBatch::INVALID_ROW_INDEX) {
+                commit_rows++;
+                output_batch->get_row(i)->set_tuple(0, dst_tuple);
+            } else {
+                // input_batch and output_batch's capacity must be same.
+                DCHECK(false);
```

Comment:
You should add ERROR info for this DCHECK.
Like: DCHECK(false) << "input_batch and output_batch's capacity must be same. " << input size << '', " << output size;